docs/config/extras.md

```diff
@@ -27,7 +27,7 @@ As of Stencil v3, several of these fields are deprecated, and will be removed in
 Deprecated fields are marked as such.
 :::
 
-Example `extras` config when __supporting__ legacy browsers:
+Example `extras` config when **supporting** legacy browsers:
```

Comment:
I just realized that vscode did these changes itself - is there something like prettier for .md files? :D

Reply:
Getting prettier in the codebase is a side project I've been working on. I'm guessing the VS Code plugin got a hold of this and decided to update it (and a few other places). The changes aren't too extensive, so I don't see a ton of harm in leaving them as-is.